reconcile/utils/ocm.py

```diff
@@ -633,12 +633,35 @@ def get_kafka_clusters(self, fields=None):
                         for cluster in clusters]
         return clusters
 
+    def get_kafka_service_accounts(self, fields=None):
+        """Returns details of the Kafka service accounts """
+        results = []
+        api = '/api/managed-services-api/v1/serviceaccounts'
+        service_accounts = self._get_json(api)['items']
+        for sa in service_accounts:
+            sa_id = sa['id']
+            id_api = f'{api}/{sa_id}'
+            sa_details = self._get_json(id_api)
+            item = {k: v for k, v in sa_details.items() if k in fields}
+            results.append(item)
+        return results
+
     def create_kafka_cluster(self, data):
         """Creates (async) a Kafka cluster """
         api = '/api/managed-services-api/v1/kafkas'
         params = {'async': 'true'}
         self._post(api, data, params)
 
+    def create_kafka_service_account(self, name):
+        """ Creates a Kafka service account """
+        api = '/api/managed-services-api/v1/serviceaccounts'
```

Comment:
This looks like a constant we can share across the entire class, doesn't it?

Reply:
i usually like copying the api directly from https://api.openshift.com, but we can go with constants.